src/main/java/org/broadinstitute/hellbender/tools/sv/PrintCNVCounts.java

```diff
@@ -0,0 +1,238 @@
+import htsjdk.samtools.SAMReadGroupRecord;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMTextHeaderCodec;
+import htsjdk.samtools.util.RuntimeIOException;
+import htsjdk.tribble.Feature;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.ExperimentalFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariantDiscoveryProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.metadata.SampleLocatableMetadata;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.SimpleCount;
+import org.broadinstitute.hellbender.utils.codecs.copynumber.SimpleCountCodec;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * <p>Prints (and optionally subsets) an rd (DepthEvidence) file or a counts file
+ * as one or more (for multi-sample DepthEvidence files) counts files for CNV determination.</p>
+ *
+ * <h3>Inputs</h3>
+ *
+ * <ul>
+ *     <li>
+ *         A locus-sorted DepthEvidence file (name ends with ".rd.txt", ".rd.txt.gz", or "rd.bci"),
+ *         or a counts file (name ends with ".counts.tsv", or ".counts.tsv.gz").
+ *     </li>
+ * </ul>
+ *
+ * <h3>Output</h3>
+ *
+ * <ul>
+ *     <li>
+ *         A counts file (or files) for use by one of the CNV-calling tools.
+ *     </li>
+ * </ul>
+ * <p>The output files are named "{output-prefix}{sample-name}.counts.tsv"</p>
+ *
+ * <h3>Usage example</h3>
+ *
+ * <pre>
+ *     gatk PrintCNVCounts \
+ *       -F input.rd.txt.gz \
+ *       -L chr1
+ * </pre>
+ *
+ * @author Ted Sharpe &lt;tsharpe@broadinstitute.org&gt;
+ */
+@CommandLineProgramProperties(
+        summary = "Prints count files for CNV determination.",
+        oneLineSummary = "Prints count files for CNV determination.",
+        programGroup = StructuralVariantDiscoveryProgramGroup.class
+)
+@ExperimentalFeature
+public class PrintCNVCounts extends FeatureWalker<Feature> {
+    public static final String INPUT_ARGNAME = "input-counts";
+    public static final String OUTPUT_PREFIX_ARGNAME = "output-prefix";
+    public static final String OUTPUT_FILES_ARGNAME = "output-files";
+
+    @Argument(
+            doc = "Input file of counts",
+            fullName = INPUT_ARGNAME,
+            shortName = StandardArgumentDefinitions.FEATURE_SHORT_NAME
+    )
+    private GATKPath inputPath;
+
+    @Argument(
+            doc = "Output file(s) prefix",
+            fullName = OUTPUT_PREFIX_ARGNAME,
+            optional = true
+    )
+    private String outputPrefix = "";
+
+    @Argument(
+            doc = "Output file containing a list of output files",
```

Comment:
Can you actually make this a tab-delimited table where the first column is sample ID and second column is the file path? This may make life easier down the road for pipelining.